src/python/py/models/quantized_model.py

```diff
@@ -339,10 +351,39 @@ def __init__(self, quant_type, input_path, bits, group_size, q_size, kv_size, in
         # Set properties of each layer based on quantization type
         self.set_properties()
 
+    def _initialize_quantized_lm_head(self, bits, group_size):
+        """
+        Initialize `QuantizedTensorModule` for LM head if not already set
+        """
+        if isinstance(self.lm_head, TensorModule):
+            assert self.lm_head.weight is None
+            assert self.lm_head.bias is None
+        if not isinstance(self.lm_head, QuantizedTensorModule):
+            self.lm_head = QuantizedTensorModule(bits, group_size)
+
     def set_properties(self):
         """
         Set in_features, out_features, and g_idx based on quantization type
         """
+        if isinstance(self.lm_head, QuantizedTensorModule):
+            if self.quant_type != "awq":
+                raise NotImplementedError(
+                    "lm_head quantization is only supported for awq."
```

Comment:
Can we set the properties for GPTQ-quantized LM head as well? It should be similar to setting the properties for AWQ-quantized LM head.